ivy/array/norms.py

```diff
@@ -19,6 +19,24 @@ def layer_norm(
         *,
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
+        """
+        ivy.Array instance method variant of ivy.layer_norm. This method simply wraps 
+        the function, and so the docstring for ivy.layer_norm also applies to this
+        method with minimal changes.
+
+        Examples
+        --------
+        >>> arr = ivy.array([[0.0976, -0.3452,  1.2740], \
```

Comment:
small detail, but let's use the variable name `x`, to be consistent with other examples.